shell/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -206,14 +204,14 @@ private static boolean validateRemote(List<String> nodes, String target, String
   private static boolean validateRemote(String node, String target, String name, CommandLine cmd)
       throws InterruptedException {
     System.out.format("Validating %s environment on %s...%n", target, node);
-    if (!Utils.isAddressReachable(node, 22)) {
+    if (!CommonUtils.isAddressReachable(node, 22)) {
       System.err.format("Unable to reach ssh port 22 on node %s.%n", node);
       return false;
     }
 
     // args is not null.
     String argStr = String.join(" ", cmd.getArgs());
-    String homeDir = ServerConfiguration.get(PropertyKey.HOME);
+    String homeDir = InstancedConfiguration.defaults().get(PropertyKey.HOME);
```

Comment:
pass in conf object

Reply:
This class is executed via `main()` so we have to instantiate conf. Saved the `InstancedConf` in an `AlluxioConf` in a field of this class.